### PR TITLE
[2] Product images loaded

### PR DESCRIPTION
**Pull request by tommypoa**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Images are now pulled from Airtable and loaded into `ProductCard` and showing on `ProductDetailsScreen`

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
- Dependent on #34.
- @thumn 's  on clerks!
## How to review

[//]: # "The order in which to review files and what to expect when testing locally"
- Make sure functionality is tested and code quality is good.

## Tests Performed, Edge Cases

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"
- ProductCard buttons are still working as it was.
### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"
image

CC: @thumn  @kennethlien 

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: lib/mapUtils.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ function createProductData(record) {
     id: data.id,
     category: data.Category,
     points: data.Points,
-    customerCost: data['Customer Cost']
+    customerCost: data['Customer Cost'],
+    image: data.Image ? data.Image[0].url : null
```
Comment (wangannie) [inline lib/mapUtils.js]: This is much cleaner than the full if/else! We should update Clerk to do this too @thumn

Comment (thumn) [reply lib/mapUtils.js]: sounds good!
